src/python/PSetTweaks/PSetTweak.py

```diff
@@ -421,6 +421,16 @@ def unpersist(self, filename, formatting=None):
             for param, value in viewitems(jsoniser.parameters):
                 self.addParameter(param , value)
 
+    def clear(self):
```

Comment:
Given that this method deletes the old tweak object and recreates a default one, shall we rename it to `reset` instead?